src/subscript/params2csv/params2csv.py

```diff
@@ -68,18 +138,30 @@ def get_parser() -> argparse.ArgumentParser:
         help="Write back cleaned parameters.txt",
         default=False,
     )
+    parser.add_argument("-v", "--verbose", action="store_true", help="Be verbose")
     parser.add_argument(
         "--version",
         action="version",
         version="%(prog)s (subscript version " + __version__ + ")",
     )
+
     return parser
 
 
-def main() -> None:
-    """Entry point from command line"""
-    parser = get_parser()
-    args = parser.parse_args()
+def params2csv_main(args: argparse.Namespace) -> None:
+    """A main function to be used both from the command line, and
+    when used as an ERT plugin (ERT workflow).
+
+    Args:
+        args (argparse.Namespace): Namespace with command line arguments
+    """
+    if args.verbose:
+        logger.setLevel(logging.INFO)
+
+    # Expand wildcards if not being expanded
+    args.parameterfile = [
+        path for pattern in args.parameterfile for path in sorted(glob(pattern))
```

Comment:
More curious than anything, but does sorting the globbed files cause any differences?

Reply:
I don't think it matter in this case. I was trying to replicate the unix glob behavior which seems to be sorted by creating time and to some level in order (probably because ERT created these parameters.txt in sequential)

Also, feels like it's nicer to see them sorted in the csv file :smile: